### PR TITLE
Add support for wrapped messages and custom popup sizes

### PR DESCRIPTION
**Pull request by Damon3000s**
- Sometimes you want to display a message to the user but don't require any additional feedback. That is what MessageOK is for. The problem is when you need to display a LOT of information to the user where the interface doesn't work.
- MessageOK only renders text unformatted which means any line of text which is too wide for the popup will be clipped making it unreadable.
- Added additional functionality to the Popup classes so we can specify that text should be displayed wrapped, and the size of the popup.

### CODE REVIEW

File: ImGuiPopupsDemo/ImGuiPopupsDemo.cs [hunk 1]
```diff
@@ -58,6 +59,35 @@ private void OnTick(float dt)
...
+
+""Sed ut perspiciatis, unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam eaque ipsa, quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt, explicabo. Nemo enim ipsam voluptatem, quia voluptas sit, aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos, qui ratione voluptatem sequi nesciunt, neque porro quisquam est, qui dolorem ipsum, quia dolor sit amet consectetur adipisci[ng] velit, sed quia non numquam [do] eius modi tempora inci[di]dunt, ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit, qui in ea voluptate velit esse, quam nihil molestiae consequatur, vel illum, qui dolorem eum fugiat, quo voluptas nulla pariatur?
+
+Cicero’s “De Finibus Bonorum et Malorum” – 45 BC""";
+			#endregion
+
+			popupMessageOK.Open("Wall Of Text", message, true, new Vector2(ImGui.GetContentRegionAvail().X * 0.75f, -1.0f));
```
Comment (matt-edmondson) [inline ImGuiPopupsDemo/ImGuiPopupsDemo.cs:88]: I would prefer not to use a bool as a parameter here, consider an enum to improve readability.

**Review by matt-edmondson — CHANGES_REQUESTED** [review]
Please add custom size support to the other uses of Modal to keep the feature set consistent across the popup types.

**Comment by Damon3000s** [comment]
Feedback addressed.

**Review by matt-edmondson — APPROVED** [review]
thanks damo

**Pull request merged**